docs/attributes-registry/dns.md

```diff
@@ -10,8 +10,9 @@
 
 This document defines the shared attributes used to report a DNS query.
 
-| Attribute           | Type   | Description                 | Examples                              | Stability                                                        |
-| ------------------- | ------ | --------------------------- | ------------------------------------- | ---------------------------------------------------------------- |
-| `dns.question.name` | string | The name being queried. [1] | `www.example.com`; `opentelemetry.io` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Attribute           | Type     | Description                                  | Examples                                                  | Stability                                                        |
```

Comment:
We can also add `error.type` here.

Reply:
it's registry section for attributes in `dns` namespace, we have `error.type` on the corresponding spans.